### ISSUE TITLE
kong:0.11.2 doesn't have kong

### ISSUE THREAD

**Issue by optii** [Kong/docker-kong#145]
Hi,

reopening a issue as this one is not resolved: https://github.com/Kong/docker-kong/issues/144

```
docker run -ti --rm kong:0.11.2 bash
Unable to find image 'kong:0.11.2' locally
0.11.2: Pulling from library/kong
5e35d10a3eba: Already exists 
eaea1d1a065c: Pull complete 
8fa0dbcc3a61: Pull complete 
Digest: sha256:0fc4dfaff0ae6618fdf3574b369cc803d0e7768792aaceee5e819d7995f65e79
Status: Downloaded newer image for kong:0.11.2

[root@733f673e7ed8 /]# ls /usr/local/
bin  etc  games  include  lib  lib64  libexec  sbin  share  src

[root@733f673e7ed8 /]# kong
bash: kong: command not found

```

kong isn't there...

**Comment by thibaultcha** [comment]
@optii Indeed. #144 was for 0.12, but I did reproduce this locally for this tag as well. Sorry about this, there was some changes made to the CentOS rpm repository and it seems like those older tags were not updated to take it into consideration. Thank you for the report, we'll provide a fix asap but it might take a few hours for Docker to deploy it - stay tuned.

**Comment by thibaultcha** [comment]
Opened at: https://github.com/docker-library/official-images/pull/4104

In the meantime, you can build it locally from aa9ab06921a36079eb362229c9241ef9b3cf894e:
```
$ cd docker-kong
$ docker build -t kong:0.11.2 centos
```

We know this is less than ideal, sorry again for not noticing the 0.11 breakage...

**Comment by thibaultcha** [comment]
The PR has been merged and the 0.11.2 tag now has Kong in it (as it should...). Thanks a lot for reporting this. There was a change to the RPM repository and we did not realize that the 0.11 image would also be rebuilt when pushing the 0.12 one - and unfortunately we did not test it. In the future, we have decided to enforce a stricter testing of both new and pre-existing docker images to avoid this from happening again.

Sorry for the inconvenience, I will be closing this now.
